yarn.lock

```diff
@@ -3691,6 +3691,36 @@
     use-resize-observer "^9.0.2"
     zustand "^3.5.2"
 
+"@stoplight/mosaic@^1.44.3":
```

Comment:
we probably don't want 2 versions of Mosaic.
We may need to bump it somewhere to avoid that